packages/dd-trace/src/plugins/database.js

```diff
@@ -4,6 +4,61 @@ const StoragePlugin = require('./storage')
 
 class DatabasePlugin extends StoragePlugin {
   static get operation () { return 'query' }
+
+  constructor (...args) {
+    super(...args)
+    this.serviceTags = {
+      dddbs: '',
+      encodedDddbs: '',
+      dde: '',
+      encodedDde: '',
+      ddps: '',
+      encodedDdps: '',
+      ddpv: '',
+      encodedDdpv: ''
+    }
+  }
+  propagationHelper (serviceTag, encodeATag, spanConfig) {
```

Comment:
This could have a more descriptive name, and should probably also be a verb as it's a method.

Reply:
updated `propagationHelper` to `encodingServiceTags`